c2corg_ui/static/js/searchdirective.js

```diff
@@ -0,0 +1,184 @@
+goog.provide('app.SearchController');
+goog.provide('app.searchDirective');
+
+goog.require('app');
+goog.require('app.utils');
+goog.require('ngeo.searchDirective');
+
+
+/**
+ * The directive for the auto-complete search field shown in the header
+ * of every page.
+ *
+ * @return {angular.Directive} Directive Definition Object.
+ */
+app.searchDirective = function() {
+  return {
+    restrict: 'E',
+    controller: 'AppSearchController',
+    bindToController: true,
+    controllerAs: 'ctrl',
+    template:
+        '<input type="text" placeholder="search…" ' +
```

Comment:
Yep. I think that can be done using ``placeholder="{{'Search...'|translate}}"``.
See for instance: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/waypoint/edit.html#L75

We also have to find a way the string detected by the angular-gettext extractor. I think that if you had a partial in https://github.com/c2corg/v6_ui/tree/master/c2corg_ui/static/partials it will be automatically detected.